src/utils/bootstrap/bootstrap.service.ts

```diff
@@ -84,6 +84,15 @@ export class BootstrapService {
         );
       const entries = Object.entries(config);
       for (const [key, value] of entries) {
+        if (
+          key === 'loggingNoPII' &&
+          value === false &&
+          process.env.AUTH_AAD_LOGGING_PII === 'false'
```

Comment:
is it not that the case we are overriding is if the env var is set to true? Also looks like code would still set the value...

Reply:
The env var is true when we are logging PII, and the key is true when no PII is logged. It's inversed logic.

Reply:
loggingNoPII has the opposite value of  process.env.AUTH_AAD_LOGGING_PII - that's what we check. Whether the hardcoded value is different than the env variable.